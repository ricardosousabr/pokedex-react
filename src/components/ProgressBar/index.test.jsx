/**
 * @jest-environment jsdom
 */

import React from "react";
import { render } from "@testing-library/react";
import BarStatus from "./index";

it("renders homepage unchanged", () => {
  const { container } = render(<BarStatus />);
  expect(container).toMatchSnapshot();
});
