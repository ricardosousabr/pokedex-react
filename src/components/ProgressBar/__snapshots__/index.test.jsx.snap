// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <progress
    class="styles__BarStatus-sc-1e9ujot-0 eKRLMZ"
    max="100"
  />
</div>
`;
